aexpect/remote_door.py

```diff
@@ -0,0 +1,718 @@
+"""
+
+SUMMARY
+------------------------------------------------------
+Share code between remote locations.
+
+Copyright: Intra2net AG
+
+
+CONTENTS
+------------------------------------------------------
+
+Utility for sharing code among multiple remote platforms, executing parts of a
+program on different locations (distributed programming) or writing controls
+executed on another host (metaprogramming).
```

Comment:
I'd have a few sentences noting that these are meant to be testing utilities and should not be enabled in production machines for security reasons.

Reply:
Sure, I added this:
```diff
@@ -42,6 +42,10 @@ We use the following three options to communicate code among remote hosts:
    persistently over the network by extending the usual classes with ones
    that are more serialization compatible (using pyro proxy instances).
 
+Note that this utility is meant to be a testing utility and should not be
+enabled in production machines for security reasons. It does contain a more
+secure approaches based on non-pickle serialization but it does open a door
+of possibilities on a remote system if not fully understood and used with care.
 
 INTERFACE
 ------------------------------------------------------
```